### PR TITLE
Coder work

### LINKED ISSUE #61: Svelte 5 support per component
- [x] QAvatar
- [x] QBreadcrumbs
- [x] QBtn
- [x] QCard
- [x] QCheckbox
- [x] QChip
- [x] QCodeBlock
- [x] QDialog
- [x] QDrawer
- [x] QFooter
- [x] QHeader - Will mostly need API change
- [x] QIcon
- [x] QInput
- [x] QLayout
- [x] QList
- [x] QCircularProgress
- [x] QLinearProgress
- [x] QRadio
- [x] QRailbar
- [x] QSelect
- [x] QSeparator
- [x] QTable
- [ ] QTabs
- [x] QToggle
- [ ] QToolbar - Only partially, will mostly need API change
- [ ] QTooltip

### PR DESCRIPTION
**Pull request by therealcoder1337**
## PR Type

> What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## What's new?

> List the changes

- Svelte 5 refactors
- Replace `$Quaff` with `Quaff`, add preliminary `router` prop (likely needs more work)
- Smaller fixes

## Screenshots

> If needed, you can add screenshots here

N/A

## This pull request closes an issue

> Add `Closes`, `Fixes` or `Resolves` followed by `#xxx[,#xxx]` where "xxx" is the issue number

Closes #61

### CODE REVIEW

File: src/lib/classes/Quaff.svelte.ts [hunk 1]
```diff
@@ -32,6 +36,10 @@ class Quaff {
       set: (newVal: boolean) => this.setDarkMode(newVal),
     };
   }
+
+  public get router() {
```
Comment (KilDesu) [inline src/lib/classes/Quaff.svelte.ts:40]: It'll do for now but this function is useless as is. Svelte 5 will most likely introduce a function to handle routing (like today's page store)

File: src/lib/components/tabs/QTab.svelte
Comment (KilDesu) [inline src/lib/components/tabs/QTab.svelte:1]: I said I was going to refactor Qtab but anyway

File: src/lib/components/tooltip/QTooltip.svelte [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-  export { userClasses as class };
+  let { value, position = "bottom", children, ...props }: QTooltipProps = $props();
+
+  Q.classes("q-tooltip", {
+    bemClasses: {
+      active: value,
+      top: position === "top",
+      right: position === "right",
+      bottom: position === "bottom",
+      left: position === "left",
```
Comment (KilDesu) [inline src/lib/components/tooltip/QTooltip.svelte:12]: You can change this to `[position]: true`

**Review by KilDesu — CHANGES_REQUESTED** [review]
Thanks for your work! Only one change requested (the `Q.classes` comment)

File: src/lib/components/tabs/QTab.svelte
Comment (therealcoder1337) [reply src/lib/components/tabs/QTab.svelte:1]: What's done is done. Feel free to change it later. :)

**Pull request merged**